services/assets/src/_post-paginate-links.scss

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
this file has indentation set at 2 spaces, the rest of the project uses 4. 
Can we update this file so to match it?